fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryIndexPlan.java

```diff
@@ -182,14 +186,12 @@ protected void appendScanDetails(StringBuilder str) {
     }
 
     @Nonnull
-    private String getRange() {
-        String range;
+    private Optional<TupleRange> getTupleRangeOptional() {
```

Comment:
I think that it is more idiomatic in the Record Layer to return a nullable `TupleRange` rather than a non null `Optional<TupleRange>`. With IntelliJ's inspections, we don't even lose the static analysis.

Reply:
done